package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dfinity/nns",
-  "version": "0.2.2",
```

Comment:
Better to 0.3.0, and check the CHANGELOG.md

We did breaking changes, therefore it makes sense to have a minor upgrade

Reply:
I believe that this PR should be closed in favour of https://github.com/dfinity/nns-js/pull/72 and the 0.2.3 release yanked.  Does that sound OK to you?

Reply:
Yes, I agree